Added support for PHP 8.4 (#3067)
ezsystems/developer-documentation: docs/getting_started/install_ibexa_dxp.md

        composer create-project ibexa/commerce-skeleton .
        ```

> ??? note "Using PHP 8.4, 8.2 or older"

Reviewer: ```suggestion
??? note "With PHP 8.4 or PHP 8.2-"
```

Author: Not a fan of `PHP 8.2-`, I don't think it's clear (for me it means 8.2 or higher, to be honest).

How about (keeping the "Using PHP..." format to be consistent with the first tab):

```suggestion
??? note "Using PHP other than 8.3 (8.4, 8.2, or older)"
```
?

Author: ```suggestion
??? note "Using PHP versions other than 8.3"
```
Or like this?

>     If you're using PHP 8.4, 8.2 or any older version, use a different set of commands:

Reviewer: ```suggestion
    If you aren't using PHP 8.3 but are using PHP 8.4, PHP 8.2 or any older version, use a different set of commands:
```

Author: Ok, but Vale will hunt me for the Oxford comma:

```suggestion
    If you aren't using PHP 8.3 but are using PHP 8.4, PHP 8.2, or any older version, use a different set of commands:
```
